CONTRIBUTING.md

```diff
@@ -39,19 +39,37 @@ This subfolder contains some example `.rst` files that show how to implement spe
 ------
 ## Releases
 
-1. Bump the version:
-   1. `git pull main`
-   2. Create a new branch
+We use semantic versioning. When starting a new minor release series like `1.11`, we create a new Git branch `2.11` and then selectively cherry-pick any bug fixes from `main` into that release.
+
+So, the release process changes whether you are releasing the first `rc1` for a release series vs. releasing later versions of the series.
```

Comment:
I think I'm a bit confused about what you mean here by the first rc1 and later versions. How do rc2 or rc3s fit into this model? Perhaps a different way of laying this out is 2 have 2 subsections like this:

<details>
<summary>Process for Minor Releases</summary>
<br>
the step by step process you currently have, but not including the rc1 stuff
</details>
<details>
<summary>Process for Release Candidates</summary>
<br>
the step by step process you currently have, but not including the regular release stuff. Or maybe a note like "follow step 1-5 of the regular release process then do xyz"
</details>